odrive-motor/src/odriveCAN.py

```diff
@@ -230,7 +228,7 @@ async def surface_errors(self):
                 errors = db.decode_message('Heartbeat', msg.data)['Axis_Error']
                 if errors != 0x0:
                     await self.stop()
-                    LOGGER.error("axis:", ODriveError(errors).name)
+                    LOGGER.error("axis:", ODriveError(errors))
```

Comment:
is .name causing any issues or masking errors in this case?

Reply:
yeah name was causing issues. it wasn't saying the actual error it was logging `axis: ODriveError(errors).name`